### ISSUE TITLE
Automate an upload to inventorypresser.com & a run of the `wp parser create` call

### ISSUE THREAD

**Issue by csalzano** [fridaysystems/inventory-presser#57]
I want the online code reference to get new versions automatically. If I could git push to make that happen with an action, I'd like it.

Bonus: also automate the plugin-zipper from Windows and the equivalent mac command to always have ready zips.

**Comment by csalzano** [comment]
1. Write some code that deletes doc pages for removed methods, functions, hooks, etc. Might need to keep track of all IDs updated & created and delete everything not contained in those lists.
2. Does code from #1 just run on the site? Probably
3. Github action does...
  1. Takes this plugin & updates the one on inventorypresser.com
  2. Runs wp-parser

**Comment by csalzano** [comment]
main.yml file maybe
`# This is a basic workflow to help you get started with Actions

name: CI

# Controls when the workflow will run
on:
  # Triggers the workflow on push or pull request events but only for the master branch
  push:
    branches: [ master ]

  # Allows you to run this workflow manually from the Actions tab
  workflow_dispatch:

# A workflow run is made up of one or more jobs that can run sequentially or in parallel
jobs:
  # This workflow contains a single job called "deploy-dot-com"
  # It deploys the plugin to inventorypresser.com
  deploy-dot-com:
    # The type of runner that the job will run on
    runs-on: ubuntu-latest

    # Steps represent a sequence of tasks that will be executed as part of the job
    steps:
      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
      - uses: actions/checkout@v2

      # Deploy to WP Engine
      - name: Deploy WordPress to WP Engine
        # You may pin to the exact commit or the version.
        # uses: wpengine/github-action-wpe-site-deploy@78e54ef1e8ce6591267521095a03e304a2b837f1
        uses: wpengine/github-action-wpe-site-deploy@main
        with:
          # The WP Engine environment to deploy to that is labeled as Production, Staging or Development.
          WPE_ENV_NAME: inventorypress
          # The public RSA key you save in the WP Engine User Portal
          WPE_SSHG_KEY_PUBLIC: ${{ secrets.WPE_SSHG_KEY_PUBLIC }} 
          # The private RSA key you will save in the Github Secrets
          WPE_SSHG_KEY_PRIVATE: ${{ secrets.WPE_SSHG_KEY_PRIVATE }} 
          # An optional source directory to deploy other than the root directory that is being versioned.
          TPO_SRC_PATH: # optional
          # An optional destination directory to deploy to other than the WordPress root.
          TPO_PATH: /wp-content/plugins/inventory-presser`

**Comment by csalzano** [comment]
Moving Phase 2 to a new issue for a better title
